frontend/apps/picasso/defi/polkadot/Assets.ts

```diff
@@ -44,8 +45,9 @@ export const Assets: {
     icon: "/tokens/usd-coin-usdc.svg",
     name: "K-USD",
     supportedNetwork: {
-      karura: null,
-      picasso: 129,
+      karura: 129,
+      picasso: 300_000_000_001, // After creating the asset id via assetRegistry, this value could be anything.
```

Comment:
so we have click up task for this in future

Reply:
Yes, once https://app.clickup.com/t/2kp9ucn is done, we need to come back and tackle this.